src/axolotl/utils/bench.py

```diff
@@ -69,8 +77,11 @@ def gpu_memory_usage_smi(device=0):
 
 
 def log_gpu_memory_usage(log, msg, device):
+    cur_device = get_device()
     if torch.backends.mps.is_available():
         usage, cache, misc = mps_memory_usage_all()
+    elif "npu" in str(cur_device):
+        usage, cache, misc = npu_memory_usage_all(device)
```

Comment:
Should we use `is_torch_npu_available` here too? We won't need to change the other lines too

Reply:
Yeah, I missed checking torch-npu availability here. It will be added in the following commit